src/main/kotlin/com/github/avrokotlin/avro4k/schema/SchemaFor.kt

```diff
@@ -37,14 +38,33 @@ interface SchemaFor {
       val BooleanSchemaFor: SchemaFor = const(SchemaBuilder.builder().booleanType())
    }
 }
+
 @ExperimentalSerializationApi
-class EnumSchemaFor(private val descriptor: SerialDescriptor) : SchemaFor {
+class EnumSchemaFor(
+   private val descriptor: SerialDescriptor
+) : SchemaFor {
    override fun schema(): Schema {
       val naming = RecordNaming(descriptor)
       val entityAnnotations = AnnotationExtractor(descriptor.annotations)
       val symbols = (0 until descriptor.elementsCount).map { descriptor.getElementName(it) }
-      val enumSchema = SchemaBuilder.enumeration(naming.name).doc(entityAnnotations.doc()).namespace(naming.namespace).symbols(*symbols.toTypedArray())
+
+      val defaultSymbol = entityAnnotations.enumDefault()?.let { fieldDefault ->
+         if (fieldDefault != Avro.NULL) {
```

Comment:
Avro.NULL is no valid default for an enum default, therefore the value within the annotation should always match a enum symbol.

Reply:
Thanks removed. Or forgot the check the actual implementation or this was earlier checking those field defaults that isn't true anymore. And changed the local variable to enumDefault to be more correct